### PR TITLE
Adds a few holidays to the holidays datum

### PR DESCRIPTION
**Pull request by Incoming5643**
Adds station prefixes for a few holidays

Holidays will now actually happen on their relevant days again, and not over a long period of time that kind of misses the point (seriously valentines day for a week? What?)

Space Station 13's birthday will now actually announce how old the game is

### CODE REVIEW

**Comment by Cheridan** [comment]
I think it's good to have 2-3 days for major holidays, it helps to get people in the spirit and we code a lot of content for stuff like Christmas and Halloween for it to just get used on that one day, especially Christmas where people are often away from home.

Stores celebrate Christmas starting at Thanksgiving, after all :^)

**Comment by Incoming5643** [comment]
Most major holidays still naturally get two with this, like Christmas still gets eve and day, Halloween doesn't just spontaneously end at midnight. Only other holiday we have coded stuff for is Easter, which still gets three because whoever coded it put in some effort to properly place Easter and I felt bad.

Those are the only holidays that really have in game effects.

**Comment by Niknakflak** [comment]
These are pretty hilarious.  Maybe someone will actually see SS13s 96th birthday

File: code/modules/holiday/holidays.dm [hunk 1]
```diff
@@ -78,7 +82,22 @@
 	begin_month = FEBRUARY
 
 /datum/holiday/birthday/greet()
-	return "Say 'Happy Birthday' to Space Station 13, first publicly playable on February 16th, 2003!"
+	var/game_age = text2num(time2text(world.timeofday, "YY")) - 3
```
Comment (Iamgoofball) [inline code/modules/holiday/holidays.dm]: > -3

File: code/modules/holiday/holidays.dm [hunk 2]
```diff
@@ -162,6 +200,25 @@
...
+		if(yy/4 == round(yy/4)) //Note: Won't work right on September 12th, 2200 (at least it's a Friday!)
+			if(dd == 12)
+				return 1
+		else
+			if(dd == 13)
+				return 1
+	return 0
+
+/datum/holiday/programmers/getStationPrefix()
+	return pick("span>","DEBUG: ","null","/list","EVENT PREFIX NOT FOUND") //Portability
```
Comment (Iamgoofball) [inline code/modules/holiday/holidays.dm]: No colons?

Comment (Incoming5643) [inline code/modules/holiday/holidays.dm]: I don't think the average player would get why that's shitcode

File: code/modules/holiday/holidays.dm (hunk 1 above)
Comment (Incoming5643) [inline code/modules/holiday/holidays.dm]: first playable version was in 2003, game age is only 0 to 99, so you take the three years after Y2K but before spessmen away from the year to get an age #lifehacks

File: code/modules/holiday/holidays.dm (hunk 2 above)
Comment (Cheridan) [inline code/modules/holiday/holidays.dm]: Should add //TODO: to the list :)

File: code/modules/holiday/holidays.dm (hunk 1 above)
Comment (Iamgoofball) [inline code/modules/holiday/holidays.dm]: Ah

File: code/modules/holiday/holidays.dm (hunk 2 above)
Comment (Iamgoofball) [inline code/modules/holiday/holidays.dm]: And // HACKY

File: code/modules/holiday/holidays.dm [hunk 3]
```diff
@@ -67,10 +70,11 @@
 
 /datum/holiday/valentines
 	name = VALENTINES
-	begin_day = 9 //6 days early
+	begin_day = 14
```
Comment (freerealestate) [inline code/modules/holiday/holidays.dm]: it'd be nice if these started a day early for people in different timezones

Comment (optimumtact) [inline code/modules/holiday/holidays.dm]: How do you start an event early for only some people?

Comment (freerealestate) [inline code/modules/holiday/holidays.dm]: You don't. The event lasts two days to make up for the fact that it starts earlier in some timezones than in others. It already works like this, a lot of events (Christmas for example) start one day or a few days early so everybody gets to experience them when they're happening in their country.

Comment (Iamgoofball) [inline code/modules/holiday/holidays.dm]: yeah we have a suprising amount of EU players

Comment (Incoming5643) [inline code/modules/holiday/holidays.dm]: fair enough, I'll dial it back a bit. We still don't need a whole week of valentines day though.

Comment (stapler2025) [inline code/modules/holiday/holidays.dm]: WHY CAN'T YOU SHARE THE LOVE INCOMING?

File: code/modules/holiday/holidays.dm (hunk 2 above)
Comment (Aranclanos) [inline code/modules/holiday/holidays.dm]: reminds me of that erro message congratulating you for not dying after 2012

**Comment by Aranclanos** [comment]
what the fuck is thanksgiving anyways i want americucks to get out

**Comment by RemieRichards** [comment]
something about turkeys

**Comment by Aranclanos** [comment]
yeah the ottomans, what about them?

**Comment by RemieRichards** [comment]
rofl

**Comment by optimumtact** [comment]
PRAISE KEMAL ATATURK FIRST OF HIS NAME, LEADER OF GLORIOUS TURK REPUBLIC

**Comment by Jordie0608** [comment]
This good to merge or is there stuff you still want to add?

**Comment by Incoming5643** [comment]
I just need to buff back some of the holiday lengths a bit for foriegners, give me  a sec I can snipe this from the web interface.

**Comment by Incoming5643** [comment]
@Jordie0608 should be good now

**Pull request merged**